packages/react/package.json

```diff
@@ -141,6 +141,7 @@
     "@storybook/addon-links": "7.6.2",
     "@storybook/addon-storysource": "7.6.14",
     "@storybook/addon-styling": "1.3.7",
+    "@storybook/blocks": "8.0.9",
```

Comment:
Could we use the v7 version of `@storybook/blocks` since we're still on v7 of storybook? 👀